app/renderer/src/actions/Inspector.js

```diff
@@ -849,22 +879,72 @@ export function setAwaitingMjpegStream(isAwaiting) {
   };
 }
 
+export function uploadGesturesFromFile(fileList) {
```

Comment:
it would be nice to add a unit test for this method

Reply:
I have explored few options for writing unit tests for this method. But since it involved using html5 api's like file reader and translation uses `localStorage`, i tried with some mocking based solutions using `JSDOM` and it doesn't turn out well.